homeassistant/components/starlink/coordinator.py

```diff
@@ -56,7 +57,17 @@ async def _async_update_data(self) -> StarlinkData:
             except GrpcError as exc:
                 raise UpdateFailed from exc
 
-    async def reboot_starlink(self):
+    async def async_stow_starlink(self, stow: bool):
+        """Set whether Starlink system tied to this coordinator should be stowed."""
+        async with async_timeout.timeout(4):
```

Comment:
Where are we catching `asyncio.TimeoutError`?

Reply:
Looks like reboot has the same issue

Reply:
I assumed it was handled automatically, just like the coordinator one. My bad 😅